dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -91,6 +67,28 @@
 import org.junit.experimental.categories.Category;
 import org.junit.rules.ExpectedException;
 
+import javax.ws.rs.core.GenericType;
```

Comment:
Why did these imports move down? I think this leads to the issue where when we build source files end up getting modified.

Reply:
The order of my imports in intellij is java, javax, all other imports, all other static imports